src/formats/bear-bear2bk.ts

```diff
@@ -1,25 +1,81 @@
-import { DataWriteOptions, normalizePath, Notice, TFile } from 'obsidian';
+import { DataWriteOptions, normalizePath, Notice, TFile, Setting } from 'obsidian';
 import { path, parseFilePath } from '../filesystem';
 import { FormatImporter } from '../format-importer';
 import { ImportContext } from '../main';
 import { readZip, ZipEntryFile } from '../zip';
 
 type Metadata = {
+	id: string;
 	ctime?: number;
 	mtime?: number;
-	archived: boolean;
-	trashed: boolean;
+	archivedtime?: number;
+	trashedtime?: number;
 };
 
 export class Bear2bkImporter extends FormatImporter {
 	private attachmentMap: Record<string, string> = {};
+	private flattenTags: boolean = false;
+	private storeId: boolean = false;
 
 	init() {
 		this.addFileChooserSetting('Bear2bk', ['bear2bk']);
 		this.addOutputLocationSetting('Bear');
+
+		new Setting(this.modal.contentEl)
+			.setName('Flatten nested tags')
+			.setDesc(
+				'When enabled, tags will be split on slashes (/) during import.'
+			)
+			.addToggle(t => t
+				.setValue(false)
+				.onChange(async v => this.flattenTags = v)
+			);
+
+		new Setting(this.modal.contentEl)
+			.setName('Store note identifiers in front matter')
+			.setDesc(
+				'This can be useful if you refered to those identifiers elsewhere than in Bear itself.'
```

Comment:
What do you think about something like "Links will be automatically updated. Enable this if the note identifier is outside of linking between notes."